Gemfile

```diff
@@ -135,6 +135,14 @@ gem 'cose', require: false
 gem 'addressable'
 gem 'json_schemer'
 
+if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("3.1")
```

Comment:
Can these gems be used on ruby versions lower than 3.1? If so it would be better to use the same code in all cases 😃

Reply:
They can. It would be reasonable to use the non-builtins always if you wanted to. But right now, Discourse runs fine on Ruby 3.0, and this seemed like the smallest change since it does nothing until 3.1.

But yes, you could just remove the "if" block around them.

Reply:
@eviltrout wdyt? 🙂

Reply:
I kind of like the if statement for now, personally.